core/src/main/java/tech/pegasys/web3signer/core/Eth1Runner.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.core;
+
+import static tech.pegasys.web3signer.core.service.http.OpenApiOperationsId.ETH1_LIST;
+import static tech.pegasys.web3signer.core.service.http.OpenApiOperationsId.ETH1_SIGN;
+import static tech.pegasys.web3signer.core.signing.KeyType.SECP256K1;
+
+import tech.pegasys.web3signer.core.config.Config;
+import tech.pegasys.web3signer.core.service.http.handlers.LogErrorHandler;
+import tech.pegasys.web3signer.core.service.http.handlers.signing.Eth1SignForIdentifierHandler;
+import tech.pegasys.web3signer.core.service.http.handlers.signing.SignerForIdentifier;
+import tech.pegasys.web3signer.core.service.http.metrics.HttpApiMetrics;
+import tech.pegasys.web3signer.core.signing.ArtifactSignerProvider;
+import tech.pegasys.web3signer.core.signing.SecpArtifactSignature;
+
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.api.contract.openapi3.OpenAPI3RouterFactory;
+import io.vertx.ext.web.impl.BlockingHandlerDecorator;
+
+public class Eth1Runner extends Runner {
```

Comment:
Would rather we didn't extend Runner as we don't need that level of customisation and just had this as the route configuration and passed into the main runner?

Reply:
Not sure how to work around this - the subclass uses LOTS of the capabilities herein ... so would need to copy some/most of this class into the different base class - and that's not making a huge difference to code layout I don't _think_.

Reply:
Let's not worry about it. Didn't realise that it relied on some much the base Runner class.